src/cpp/session/modules/SessionGit.cpp

```diff
@@ -3354,21 +3354,12 @@ core::Error initialize()
       return error;
 
    // setup environment
-   BOOST_ASSERT(boost::algorithm::ends_with(sshAskCmd, "rpostback-askpass"));
    core::system::setenv("GIT_ASKPASS", "rpostback-askpass");
-
    if (interceptAskPass)
-   {
       core::system::setenv("SSH_ASKPASS", "rpostback-askpass");
-   }
 
    // add postback directory to PATH
-   FilePath postbackDir = session::options().rpostbackPath().getParent();
-   if (postbackDir.getAbsolutePath().find("session/postback") == std::string::npos) {
-      // for package builds only, postback/rpostback-askpass in same directory as rpostback itself
-      postbackDir = postbackDir.completeChildPath("postback");
-   }
-
+   FilePath postbackDir = module_context::rPostbackScriptsDir();
    core::system::addToPath(postbackDir.getAbsolutePath());
```

Comment:
TODO: This doesn't seem to be working on Windows (the `rsession.exe` PATH isn't updated); we might need to call `Sys.setenv()` directly?